elements/pfe-button/pfe-button.ts

```diff
@@ -0,0 +1,180 @@
+import { LitElement, html } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import { unsafeHTML } from 'lit/directives/unsafe-html.js';
+import { ifDefined } from 'lit/directives/if-defined.js';
+
+import { bound, initializer, observed, pfelement } from '@patternfly/pfe-core/decorators.js';
+import { pfeEvent } from '@patternfly/pfe-core/functions/pfeEvent.js';
+import { Logger } from '@patternfly/pfe-core/controllers/logger.js';
+
+import styles from './pfe-button.scss';
+
+/**
+ * Buttons allow users to perform an action when triggered. They feature a text label, a background or a border, and icons.
+ *
+ * @fires {Event} pfe-button:click {@deprecated use native click event instead}
+ *
+ * @cssprop {<color>} --pfe-button--BackgroundColor
+ *          no region
+ *          {@default var(--pfe-theme--color--ui-accent, #06c)}
+ * @cssprop {<color>} --pfe-button--Color
+ *          no region
+ *          {@default var(--pfe-theme--color--ui-base--text, #fff)}
+ * @cssprop --pfe-button--FontSize
+ *          no region
+ *          {@default var(--pf-global--FontSize--md, 1rem)}
+ * @cssprop {normal | bold | bolder | lighter | <number [1,1000]>} --pfe-button--FontWeight
+ *          no region
+ *          {@default var(--pfe-theme--font-weight--normal, 400)}
+ * @cssprop {<length>} --pfe-button--Padding
+ *          no region
+ *          {@default calc(var(--pfe-theme--container-padding, 1rem) / 2) var(--pfe-theme--container-padding, 1rem)}
+ * @cssprop {<length>|<percentage>} --pfe-button--BorderRadius
+ *          no region
+ *          {@default var(--pfe-theme--surface--border-radius, 3px)}
+ * @cssprop {<number>} --pfe-button--LineHeight
+ *          no region
+ *          {@default var(--pfe-theme--line-height, 1.5)}
+ * @cssprop --pfe-button__after--Border
+ *          region `after`
+ *          {@default var(--pfe-theme--ui--border-width, 1px) var(--pfe-theme--ui--border-style, solid) var(--pfe-button__after--BorderColor, transparent)}
+ * @cssprop {<color>} --pfe-button--BackgroundColor--hover
+ *          no region
+ *          {@default var(--pfe-theme--color--ui-accent--hover, #004080)}
+ * @cssprop --pfe-button__after--Border--hover
+ *          region `after`
+ *          {@default var(--pfe-theme--ui--border-width, 1px) var(--pfe-theme--ui--border-style, solid) var(--pfe-button__after--BorderColor--hover, transparent)}
+ * @cssprop --pfe-button--FontWeight--large
+ *          no region
+ *          {@default var(--pfe-theme--font-weight--semi-bold, 600)}
+ * @cssprop --pfe-button--Padding--large
+ *          no region
+ *          {@default 12px 24px}
+ *
+ * @csspart container
+ * @csspart {HTMLButtonElement} button - The shadow button
+ */
+@customElement('pfe-button') @pfelement()
+export class PfeButton extends LitElement {
+  static readonly styles = styles;
+
+  /** Disables the button */
+  @observed
+  @property({ type: Boolean }) disabled = false;
```

Comment:
I'm pretty sure this was already fixed.  But the disabled property should be set to `reflect: true`

```Suggestion
@property({ type: Boolean, reflect: true }) disabled = false;
```